spec/features/guest_spec.rb

```diff
@@ -101,14 +101,28 @@
       end
 
       it 'can remove items from cart' do
-        click_link 'Remove',
-                   href: "/remove_from_cart/#{EquipmentModel.first.id}"
+        @quantity = 0
+
+        fill_in 'quantity_field', with: @quantity.to_s
+        find(:xpath, "//input[@id='quantity_field']").set @quantity.to_s
```

Comment:
Just out of curiosity, is there a reason why xpath would be preferred here over a CSS selector?